package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "block-lab",
   "title": "Block Lab",
-  "version": "1.3.1",
+  "version": "1.3.2",
```

Comment:
I'm not sure why this change is showing up in this diff, but the version is already `1.3.2` in the [package.json](https://github.com/getblocklab/block-lab/blob/develop/package.json#L4) of the `develop` branch.